doc/api.rst

```diff
@@ -27,17 +27,29 @@ List of base classes:
 Benchopt run hooks
 ~~~~~~~~~~~~~~~~~~
 
-:func:`~benchopt.BaseSolver.skip`: hook to allow skipping some configurations.
-Also available for ``Dataset`` and ``Objective``.
+:func:`benchopt.BaseObjective.skip`: hook to allow skipping configurations.
+It is executed after ``get_data`` to skip if current objective is
+not compatible with dataset.
 
-:func:`~benchopt.BaseSolver.get_next`: hook to change the sampling points for
-a given solver.
+:func:`benchopt.BaseSolver.skip`: hook to allow skipping configurations.
+It is executed right after ``set_objective`` to skip if solver
+if not compatible with objective and/or dataset parameters.
+Refer to :ref:`Advanced usage <skiping_solver>` for an example.
```

Comment:
skiping/skipping

Reply:
it is actually the name used in the doc

https://github.com/benchopt/benchopt/blob/c0eef0082f37dd87082e86057faa7295a7a45084/doc/advanced.rst?plain=1#L63-L67

I abided by it to prevent potential side effects.
Should we rename it in this PR, @mathurinm?